src/DomManager.js

```diff
@@ -10,8 +10,27 @@ export default class DomManager {
     return this;
   }
 
-  populateAll(results, container) {
-    results.forEach((result) => this.populate(result, container));
+  appendAllResults(results, container) {
```

Comment:
- [ ] Let's use `arrow functions` instead of the standard functions as it is part of the requirements for this entire project. The goal is to learn how to use proper ES6 syntax.

- [ ] Note that as you are using classes you will not need to declare your arrow functions with `const, let or var`